engine/table/src/test/java/io/deephaven/engine/table/impl/ssa/TestLongSegmentedSortedArray.java

```diff
@@ -18,7 +18,6 @@
 import io.deephaven.engine.rowset.chunkattributes.RowKeys;
 import io.deephaven.chunk.attributes.Values;
 import io.deephaven.chunk.LongChunk;
-import io.deephaven.chunk.LongChunk;
```

Comment:
This suggests that spotless or import optimization has un-done the work of code replication. Try running the replicator and re-committing.

Reply:
yeah, it was import optimization. I rereplicated and it undid it.